benchmarks/execute/benches/fibonacci_execute.rs

```diff
@@ -1,44 +1,44 @@
-use criterion::{criterion_group, criterion_main, Criterion};
```

Comment:
commented out this file now. this was doing the old execute (e3) which now requires segments as input. we should probably delete it or benchmark e3 in some other way